configure.php

```diff
@@ -797,6 +797,34 @@ function getFileModificationHistory(): array {
     }
 }
 
+echo "Process PI nodes...\n";
+if (!$ac['LANG'] == 'en') {
+    if (file_exists($srcdir . '/pi_processing/langs/' . $ac['LANG'] . '.php')) {
+        require_once $srcdir . '/pi_processing/langs/' . $ac['LANG'] . '.php';
+    }
+}
+require_once $srcdir . '/pi_processing/langs/en.php';
+require_once $srcdir . '/pi_processing/processors.php';
+$xpath = new DOMXPath($dom);
+foreach ($xpath->query('//processing-instruction()') as $pi) {
+    if (!str_starts_with($pi->target, 'phpdoc')) {
+        continue;
+    }
+    // Don't care about generic phpdoc PI target
+    if ($pi->target === 'phpdoc') {
```

Comment:
No, because we have a bunch of PI instruction that are just `phpdoc` and those that will generate documentation node have more complicated names so the target is effectively the instruction and the date are values to pass to the corresponding function.

Maybe this design is crap, but I didn't want to figure out how to parse the function to call